rust/src/nasl/syntax/statement.rs

```diff
@@ -536,4 +544,236 @@ mod position {
 
         assert_eq!(tests, expected.len());
     }
+
+    #[test]
+    fn primitive() {
+        let code = "1;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn variable() {
+        let code = "a;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn array() {
+        let code = "a[1];";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn call() {
+        let code = "a();";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn exit() {
+        let code = "exit(0);";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn return_stmt() {
+        let code = "return 0;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn break_stmt() {
+        let code = "break;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn continue_stmt() {
+        let code = "continue;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn include() {
+        let code = "include(\"test.inc\");";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn declare() {
+        let code = "local_var a;";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn parameter() {
+        let code = "(a, b);";
+        let mut parser = parse(code);
+        let stmt = parser.next().unwrap().unwrap();
+        assert_ron_snapshot!(stmt);
+    }
+
+    #[test]
+    fn named_parameter() {
+        let code = "a: b;";
```

Comment:
Not sure, if this should be already counted as named_parameter without the brackets `(a: b)`